README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+## Yanwen Wang
+I'm **Yanwen Wang**, a second-year student at UChicago SSA. Here's my [Linkedin](https://www.linkedin.com/in/yanwen-wang-3a0329105/) page. The followings are a list of my hobbies:
+
+- _Piano:_ My favorite piece to play is "The Goldberg Variations".
+- _Photography:_ My photography style is very conceptual.
+- _Hiking:_ I summitted Mt. Kilimanjaro in 2017.
+
+## Images
+![Image of Yanwen](yanwen.jpeg)
+
+## General workflow
+I used RStudio Cloud on iPad since my laptop is under repair. Basically, after forking the project, I opened a new project with a Git Repo on RStudio Cloud, thus connecting my RStudio to Github. I made changes to the Readme.md using RMarkdown. With some previous experience using Jupyter Notebook with Python, I didn't find any difficulty when using RMarkdown. And now I'm ready to make the push request and upload it back to Github.
+
+## Difficulties
+I experienced the most difficulties during the commit and push procedure, expecially wondering how to push the md file along with an image in the working directory.
+
+Initial mistake: I set up the path of the image as /cloud/project/image, according to the directory in Rstudio, however, I should directly set the path according to Github directory.
```

Comment:
just a cool tweak you can do here:
`/cloud/project/image`
Use `` to format as a code!